### ISSUE TITLE
export and download only exports half of image

### ISSUE THREAD

**Issue by cspencer905** [publiclab/mapknitter#167]
I'm not sure if I've done something wrong with the settings but this issue happens with multiple images I've tried, apparently in the same position with each image. I've made sure the extent of the map exceeds the image and have tried locking/unlocking the photo as troubleshooting. Any thoughts?

![export issue](https://cloud.githubusercontent.com/assets/11618945/7253577/a233eafe-e835-11e4-84f7-2d812bbf3512.png)
![export issue2](https://cloud.githubusercontent.com/assets/11618945/7253578/a433d72e-e835-11e4-863e-de9b780a3517.png)

**Comment by jywarren** [comment]
Hi, what does the export itself look like? Thanks for reporting!

Also, can you link to the map so we can check if the images have any odd
metadata or anything that could be confusing MapKnitter?

**Comment by cspencer905** [comment]
http://mapknitter.org/maps/gournia

The geoTIFFS illustrated in the image above are the exports.

**Comment by cspencer905** [comment]
I have also tried another map (the current image is the second try) which end with a different error. Once placing the map, I use the export functionality (it is completed suscpiciously fast within a couple seconds). When I go to export as a geoTIFF is the drop down window this format is the only format available (no JPG, OSM, nothing). This sends me to the error page: http://mapknitter.org/export/geotiff/gournia.tif providing no further information about the error. 
This suggests the export is not actually happening.
I have tried this on both Google Chrome and Mozilla web browsers, and only appears to happen when there is only one image.
![dl-issue](https://cloud.githubusercontent.com/assets/11618945/7365943/9ac70de4-ed8c-11e4-9792-3109fcc3b537.png)

**Comment by jywarren** [comment]
Sorry we haven't been able to help out on this yet! Juggling a few different bugs at the moment. Have bumped this up onto our urgent list. Thanks for your help reporting it.

**Comment by jywarren** [comment]
Hi, it looks like you just tried exporting again, and it failed; the site also saw some bumpiness. I'm going to take a look at this now, sorry!

**Comment by jywarren** [comment]
It looks like that last attempt to export also didn't work? Hmm. It may be in part that this is just a very large-scale map; could we try turning the resolution down more? The slider doesn't go any lower, but I could tweak that too to allow, say, a 350 cm/px export?

**Comment by cspencer905** [comment]
Could the scale of the map be a reason why the first maps I tried (the first set of images in this post) also failed to export the entire image?

**Comment by jywarren** [comment]
It's possible -- say, if the machine ran out of memory or something partway
through exporting? Do you want me to try making it possible to set a lower
resolution?

**Comment by cspencer905** [comment]
Yes it would greatly appreciated. I have been having this issue with other maps I have tried to export and would be happy to know if this was the issue.

**Comment by jywarren** [comment]
I'm sorry to say I have to write the Web Working Group's monthly report
this afternoon, but will try to work on this after that. Thanks for your
patience.

**Comment by jywarren** [comment]
Hi, cspenser905 - sorry, I tried a few different things and haven't been able to get this to work. I'm going to change the name of the issue to reflect the cause we've semi-discovered. I hope you've been able to find another solution -- one I could suggest would be using GDAL (gdal.org) directly, which is what MapKnitter does anyways -- especially since your source images are already projected and you won't need any distortion, just coordinate system metadata, to form a correct GeoTiff. And `gdal2tiles.py`, also in that suite of utilities, will get you tiles etc. 

Apologies again!

**Comment by jywarren** [comment]
Actually i see you are distorting a bit -- but GDAL does that too, with provided GCPs (geographic control points). You might also try Map Warper, also based on GDAL: http://maps.nypl.org/warper
